core-android/src/main/java/com/mux/stats/sdk/muxstats/MuxStateCollector.kt

```diff
@@ -52,16 +52,23 @@ open class MuxStateCollector(
 
   /**
    * This is the time to wait in ms that needs to pass after the player has seeked in
-   * order for us to conclude that playback has actually started. We use this value in a workaround
-   * to detect the playing event when {@link SeekedEvent} event is some times reported by
-   * {@link ExoPlayer} a few milliseconds after the {@link PlayingEvent} which is not what is
-   * expected.
+   * order for us to conclude that playback has actually started. This is to ensure that callback
+   * ordering doesn't cause state issues during sometimes-chaotic player startup
    * */
   @Suppress("unused")
   var timeToWaitAfterFirstFrameReceived: Long = 50
 
   /**
-   * The current state of the player, as represented by Mux Data
+   * The current state of the player, as represented by Mux Data. Only mutable from the inside
```

Comment:
> Only mutable from the inside

Would describing as readonly be accurate?

Reply:
That exact word doesn't appear a lot, but honestly just letting it be `val` is probably enough. 

Honestly I think what I really want to say is that you need to use the methods in the `@see`s to mutate the state, so I'm going to do a little rewrite